packages/core/src/PlayerContextProvider.js

```diff
@@ -56,6 +56,8 @@ const defaultState = {
    * complete
    */
   awaitingResumeOnSeekComplete: false,
+  // true if media was playing when the active track changed
```

Comment:
I think a more accurate description would be "true if media will play once track has loaded".

Reply:
what about: true if media will play once new track has loaded?

Reply:
sure!